src/common/StyledCard/index.ts

```diff
@@ -0,0 +1,27 @@
+'use client'
+import styled from '@emotion/styled'
+import { $whiteColor, $blueColor, $pinkColor } from '../../styles/_variables'
+
+export const StyledCard = styled.div`
+  position: relative;
+  padding: 1rem;
+  box-shadow:
+    ${$pinkColor} 0px 0px 10px 2px inset,
+    ${$blueColor} 0px 0px 10px 2px;
+  border-radius: 20px;
+  inset: -1px;
+
+  &::before {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    border-radius: 20px;
+    padding: 1px;
+    background: linear-gradient(135deg, ${$blueColor}, ${$pinkColor});
+    -webkit-mask:
```

Comment:
webkit is only for Safari right? mozilla it won't work.

Reply:
What do you want from this?

Reply:
found alternate solution